graphql/introspection/introspection.go

```diff
@@ -190,7 +193,19 @@ func (s *introspection) registerType(schema *schemabuilder.Schema) {
 		}
 	})
 
-	object.FieldFunc("enumValues", func(args struct{ IncludeDeprecated *bool }) []EnumValue {
+	object.FieldFunc("enumValues", func(t Type, args struct {
+		IncludeDeprecated *bool
+	}) []EnumValue {
+
+		switch t := t.Inner.(type) {
+		case *graphql.Enum:
+			var enumVals []EnumValue
+			for _, v := range t.Values {
+				enumVals = append(enumVals,
+					EnumValue{Name: v, Description: "", IsDeprecated: false, DeprecationReason: ""})
```

Comment:
How about we include the Go value in the description?

Reply:
this seems like a nice thing to do